src/main.tsx

```diff
@@ -21,9 +20,9 @@ const queryClient = new QueryClient({
   defaultOptions: {
     queries: {
       // staleTime: Number.POSITIVE_INFINITY,
-      // retry: MAX_RETRIES,
       retry: (failureCount, error): boolean => {
-        return failureCount <= MAX_RETRIES;
+        // return failureCount <= MAX_RETRIES;
+        return 0;
```

Comment:
Function return type is `boolean`
```suggestion
        return false;
```

Reply:
Question: Does this need to be a function?  Could it just be `retry: 0`?

Looking at the docs, their [examples just pass the retry value](https://tanstack.com/query/v5/docs/framework/react/guides/query-retries)